app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -378,11 +378,7 @@ private void setUpMenu() {
     @VisibleForTesting
     public BrowserFragment getVisibleBrowserFragment() {
```

Comment:
Seems like some functions that depends on getVisibleBrowserFragment() will change behavior: for instance, when TopFragment is HomeFragment and TurboMode is switched, Site currently focused will not be reloaded. (However this sounds acceptable, lol.)